src/views/AnnualReport.vue

```diff
@@ -415,7 +417,7 @@ export default {
       return this.certifyText(FilingCodes.ANNUAL_REPORT_OT)
     },
 
-    payAPIURL () {
+    payApiUrl () {
```

Comment:
Return type string for payApiUrl, saw this updated everywhere else.

Reply:
I'll fix this in an upcoming PR.